internal/clients/agent.go

```diff
@@ -1,55 +1,61 @@
-package main
+package clients
 
 import (
 	"context"
-	"time"
-
 	pb "github.com/RHEcosystemAppEng/cluster-iq/generated/agent"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+// TOdO, review timeout
```

Comment:
WDYT about configuring 60s by default, and potentially adding a EnvVar for configuring it ?